benchmarks/golden_configs/oss_mnist.py

```diff
@@ -4,9 +4,9 @@
 def get_golden_real_stats():
 
     return {
-        "reference_speed": 1430,
-        "reference_memory": 1220,
-        "reference_loss": 0.006,
+        "reference_speed": 660,
```

Comment:
the numbers left must have been on a devfair, the number right are the ones we used on circleci previously